migrations/migrations/20201218183543-create_table_users.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
         type: Sequelize.STRING,
         allowNull: false
       },
+      is_admin: {
+        type: Sequelize.STRING,
```

Comment:
I think that here the type should be boolean. Why do you define it as string?
Apart from this, Imagine that this table is already created in production. You cant revert the create migration because this action will delete the table and all data stored in it. 
So, in this case, you must create a new migration adding the new column to the table.